draft-ietf-mandyam-eat.md

```diff
@@ -260,6 +260,111 @@ signing. In this case the manufacturer should perform the verification,
 because any release of the key material would enable a participant other than the
 entity to create valid signed EATs.
 
+## Claim Characteristics
+
+### Interoperability and Relying Party Orientation
+
+One of the broad goals for EAT is that tokens can be processed by
+relying parties in a general way regardless of the type, manufacturer
+or technology of the entity from which they originate. It is a goal
+that there be some general purpose EAT verification implementations
+that can process and verify tokens for large numbers of use cases
+without having special cases and configurations for different entity
+and device types. This is a goal of interoperability of claims
+themselves, not just of the signing format and representation syntax.
+
+This is a lofty goal and it will be difficult to achieve broadly as it
+requires careful definition of claims in a technology neutral way.
+Sometimes two entity types will have two ways of doing something that
+are largely irreconcilable. However, the goal remains, even if it is a
+very long-term goal, where entities gradually get designed so that
+they more and more easily can interoperate with EAT.
+
+### OS and Technology Neutral
+
+Claims should be defined such that they are not specific 
+to any operating system. They should be applicable to
+multiple large high-level operating systems from different
+vendors. They should also be applicable to multiple 
+embedded operating systems from multiple vendors and
+everything in between.
+
+Claims should not be defined such that they are specific to any SW
+environment or programming language.
+
+Claims should not be defined such they are not specific to any chip or
```

Comment:
```suggestion
Claims should not be defined such they are specific to a chip or
```
or
```suggestion
Claims should be defined such they are not specific to any chip or
```

Reply:
Shouldn't the language be "...such THAT they..."?

Not sure why the EAT draft should give guidance on what belongs as a claim (or not). Doesn't the existence of the claim in the spec resolve whether or not it should exist?